lib/openshift/image_pruner_imageregistry_operator_openshift_io.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ImagePrunerImageregistryOperatorOpenshiftIo < ClusterResource
+    RESOURCE = 'imagepruner.imageregistry.operator.openshift.io'
```

Comment:
I expect this still needs to be `imagepruners.imageregistry.operator.openshift.io`

Reply:
@akostadinov interestingly it works without it...but I updated it anyway.   PTAL.  Thanks!

Reply:
@pruan-rht , presently doing proper plural of the item should not break often because it would only be used with API calls. But I have not seen us do API calls for a long time. Still better to have it right in case we start doing that.